drivers/pds/lib/resiliency_utils.go

```diff
@@ -111,6 +114,26 @@ func InduceFailureAfterWaitingForCondition(deployment *pds.ModelsDeployment, nam
 			InduceFailure(FailureType.Type, namespace)
 		}
 		ExecuteInParallel(func1, func2)
+	case RebootNodesDuringDeployment:
+		checkTillReplica = CheckTillReplica
```

Comment:
Can we use directly CheckTillReplica rather using one local variable ?

Reply:
Yes true, you are right, this is slightly weird. So this is on my radar as Madan and Arun have pointed it out as well. I am pushing it to another PR but. Reason is : As of now we just have one variable here to check - checkTillReplica. In the next PR I will be introducing a new variable and then will try to handle that and this variable together (maybe through a channel or maybe through a param json - need to define a proper design to handle multiple parallel scenarios) and that's why I am keeping it as is.

Reply:
Please create a PA ticket to track the changes.

Reply:
Makes sense - [PA-867](https://portworx.atlassian.net/browse/PA-867)